### PR TITLE
Fixes floating of hero buttons on mobile

### PR DESCRIPTION
**Pull request by mdworken**
(Images of difference posted to slack)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,10 +28,10 @@
             <div class="hero-normal">
                 Dedicated to making the world <strong>gooder</strong>
             </div>
-        </div>
-        <div class="hero-bottom center">
-          <a href="https://ti.to/codeforgood/rubybythebay" class="btn btn-huge btn-3d"><i class="fa fa-heart fa-1x"></i>Register</a>
-          <a href="https://ti.to/codeforgood/rubybythebay" class="btn btn-huge btn-3d"><i class="fa fa-heart fa-1x"></i>Sponsor an attendee</a>
+            <div class="hero-bottom center">
```
Comment (mdworken) [inline index.html]: @jtu0 I moved this part into the "here-content" div. It's not *quite* the right semantic naming anymore but I think it's reasonable enough.

File: assets/css/custom.css [hunk 2]
```diff
@@ -17,8 +17,7 @@
 }
 
 #hero .hero-bottom {
-  position: absolute;
-  bottom: 75px;
```
Comment (mdworken) [inline assets/css/custom.css]: Removed this because it was no longer necessary.

File: assets/css/custom.css [hunk 3]
```diff
@@ -17,8 +17,7 @@
...
+  position: relative;
```
Comment (mdworken) [inline assets/css/custom.css]: Relative made a lot more sense here than absolute because of how the nav bar works, I think

**Comment by jtu0** [comment]
Thank you @mdworken - and thanks for the abundant commenting!

**Pull request merged**